README.md

```diff
@@ -49,7 +49,7 @@ See the package wiki for more information on [installation troubleshooting](http
 
 ## Documentation
 
-See the [igraph homepage](https://igraph.org/r/) for the complete manual.
+See the [igraph package's website](https://igraph.org/) for the complete manual.
```

Comment:
Let's not call https://igraph.org/ the "package website". That would be https://r.igraph.org/, no?